packages/commonwealth/client/scripts/views/components/react_quill_editor/use_image_drop_and_paste.tsx

```diff
@@ -72,11 +69,32 @@ export const useImageDropAndPaste = ({
           app.user.jwt
         );
 
+        const selectedIndex = editor.getSelection()?.index;
+        const text: string = editor.getText();
+        let blankEditor = false;
+        let lineBreak = '\n';
+
+        // When Quill editor is blank, getLength returns 1
+        if (editor.getLength() > 1) {
+          if (text[selectedIndex - 1] && text[selectedIndex]) {
+            lineBreak = '\n\n';
+          }
+        } else {
+          blankEditor = true;
+        }
+
         // insert image op at the selected index
         if (isMarkdownEnabled) {
           editor.insertText(selectedIndex, `![image](${uploadedFileUrl})`);
         } else {
-          editor.insertEmbed(selectedIndex, 'image', uploadedFileUrl);
+          if (!blankEditor) {
+            editor.insertText(selectedIndex, lineBreak);
+          }
+          if (selectedIndex === 0) {
+            editor.insertEmbed(selectedIndex, 'image', uploadedFileUrl);
+          } else {
+            editor.insertEmbed(selectedIndex + 1, 'image', uploadedFileUrl);
+          }
```

Comment:
please add comments in the code what cases these if statements cover

Reply:
cc @martins87

Reply:
[Addressed](https://github.com/hicommonwealth/commonwealth/pull/4109/commits/a59211db3ed988a1cbfc461f9a81bda89a9c3806).